app/src/main/res/menu/commonmenu.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/share"
+        android:icon="@drawable/ic_share_black_24dp"
+        android:title="Share"
+        app:showAsAction="ifRoom" />
```

Comment:
@Abudu-Samuel 
since you are only having `one menu item (share)`, the `showAsAction` app property should have an `always` value instead.

Reply:
Thanks for the feedback, will do the needful.